app/js/lib/foxbox/services/base.js

```diff
@@ -232,7 +237,7 @@ export default class BaseService extends EventDispatcher {
    * super.teardown() method as well.
    */
   teardown() {
-    for (let watcher of this[p.watchers].values()) {
+    for (const watcher of this[p.watchers].values()) {
```

Comment:
hmm, now when I read this code, I'm not sure how it should work in reality :) Do you think it should be `let` and eslint should not complain instead?